src/java.base/share/classes/sun/net/www/protocol/http/Negotiator.java

```diff
@@ -82,5 +82,7 @@ private static void finest(Exception e) {
             logger.finest("NegotiateAuthentication: " + e);
         }
     }
+
+    public abstract void disposeContext() throws IOException;
```

Comment:
We have some old internal tests that extend  `Negotiator` and that will fail when this change is pushed. I'd suggest to change this method as follows:

```suggestion
    public void disposeContext() throws IOException { };
```

This should avoid breaking existing subclasses that do not implement this method.

Reply:
Thank you. Changed from abstract to empty.
sun/security/jgss sun/security/krb5 sun/net/www/protocol/http tests passed